tree/ntuple/v7/test/rfield_vector.cxx

```diff
@@ -31,6 +31,51 @@ TEST(RNTuple, ClassVector)
    }
 }
 
+
+TEST(RNTuple, InsideCollection)
+{
+   FileRaii fileGuard("test_ntuple_insidecollection.root");
+
+   auto modelWrite = RNTupleModel::Create();
+   auto wrKlassVec = modelWrite->MakeField<std::vector<CustomStruct>>("klassVec");
+   CustomStruct klass;
+   klass.a = 42.0;
+   klass.v1.emplace_back(1.0);
+   klass.v1.emplace_back(2.0);
+   wrKlassVec->emplace_back(klass);
+
+   {
+      RNTupleWriter ntuple(std::move(modelWrite),
+         std::make_unique<RPageSinkFile>("myNTuple", fileGuard.GetPath(), RNTupleWriteOptions()));
+      ntuple.Fill();
```

Comment:
would there be any reason to assume that having the `std::vector` re-allocate and then calling `Fill` again might not work?

Reply:
That should work fine